po/fr.po

```diff
@@ -1,5 +1,723 @@
+
+#: src/Layouts/Partials/TransformPanel.vala:98
+msgid "Vertical position"
+msgstr "Position verticale"
+
+#: src/Layouts/Partials/TransformPanel.vala:100
+msgid "W"
+msgstr "W"
+
+#: src/Layouts/Partials/TransformPanel.vala:100
+msgid "Width"
+msgstr "Largeur"
+
+#: src/Layouts/Partials/TransformPanel.vala:102
+msgid "H"
+msgstr "H"
+
+#: src/Layouts/Partials/TransformPanel.vala:102
+msgid "Height"
+msgstr "Hauteur"
+
+#: src/Layouts/Partials/TransformPanel.vala:107
+msgid "Lock Ratio"
+msgstr "Ratio de verrouillage"
+
+#: src/Layouts/Partials/TransformPanel.vala:114
+msgid "R"
+msgstr "R"
+
+#: src/Layouts/Partials/TransformPanel.vala:114
+msgid "Rotation degrees"
+msgstr "Degrés de rotation"
+
+#: src/Layouts/Partials/TransformPanel.vala:126
+msgid "Flip Horizontally"
+msgstr "Retournement horizontal"
+
+#: src/Layouts/Partials/TransformPanel.vala:137
+msgid "Flip Vertically"
+msgstr "Retournement vertical"
+
+#: src/Layouts/Partials/TransformPanel.vala:165
+msgid "Position"
+msgstr "Position"
+
+#: src/Layouts/Partials/TransformPanel.vala:169
+msgid "Size"
+msgstr "Taille"
+
+#: src/Layouts/Partials/TransformPanel.vala:178
+msgid "Opacity"
+msgstr "Opacité"
+
+#: src/Layouts/RightSideBar.vala:111
+msgid "Search Layer"
+msgstr "Rechercher un calque"
+
+#: src/Lib/Managers/ExportManager.vala:192
+#: src/Lib/Managers/ExportManager.vala:219
+msgid "Generating preview, please wait…"
+msgstr "Génération de l'aperçu, veuillez patienter…"
+
+#: src/Lib/Managers/ExportManager.vala:313
+#, c-format
+msgid "Untitled %i"
+msgstr "Sans titre %i"
+
+#: src/Lib/Managers/ExportManager.vala:484
+msgid "Exporting images…"
+msgstr "Exporter des images…"
+
+#: src/Partials/ExportWidget.vala:117
+#, c-format
+msgid "%i × %i px · %s"
+msgstr "%i × %i px · %s"
+
+#: src/Partials/ExportWidget.vala:124
+msgid "Fetching image size…"
+msgstr "Récupération de la taille de l'image…"
+
+#: src/Partials/ZoomButton.vala:49
+msgid "Zoom Out"
+msgstr "Dézoomer"
+
+#: src/Partials/ZoomButton.vala:55
+msgid "Reset Zoom"
+msgstr "Remettre à zoom à zéro"
+
+#: src/Partials/ZoomButton.vala:62
+msgid "Zoom In"
+msgstr "Zoomer"
+
+#: src/Partials/ZoomButton.vala:68
+msgid "Zoom"
+msgstr "Zoom"
+
+#: src/Services/ActionManager.vala:217
+msgid "No recently opened file available!"
+msgstr "Aucun dossier récemment ouvert n'est disponible !"
+
+#: src/Services/ActionManager.vala:236
+msgid "No second most recently opened file available!"
+msgstr "Pas de deuxième fichier le plus récemment ouvert disponible !"
+
+#: src/Services/ActionManager.vala:255
+msgid "No third most recently opened file available!"
+msgstr "Pas de troisième fichier le plus récemment ouvert disponible !"
+
+#: src/Services/ActionManager.vala:293
+msgid "Nothing selected to export!"
+msgstr "Rien de sélectionné pour l'exportation !"
+
+#: src/Services/ActionManager.vala:302
+msgid "Export of Artboards currently unavailable…sorry 😑️"
+msgstr "Exportation de plans de travail actuellement indisponible… désolé 😑️"
```

Comment:
This should be either
> L'export des plans de travail est actuellement indisponible... pardon 😑️

or
> L'export des espaces de travail est actuellement indisponible... pardon 😑️

depending on the translation choice about `artboard`